clipboard-apis/cancelling_default_action_basic_test_copy_on_inputtext.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html><head>
+    <meta charset="UTF-8">
+    <title>Clipboard API - cancelling default action, basic test   - copy on 'input_text'</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="support/_lib.js"></script>
+    <script>
+/** cancelling default action, basic test   */
+/* Events: copy cut */
+/* Test HTML: <form><input id="input_text" autofocus onfocus="this.select()" value="copied text"></form>*/
+/* Targets: document 'input_text' */
+/* paste data: "clipboard text" */
+/* External pass condition - clipboard data: "clipboard text" */
+function clipboard_api_test(e){
+    e.preventDefault();
+}
+
+    </script>
+</head>
+<body>
+
+    <p>FAILED (This TC requires JavaScript enabled)</p>
+    <form><input id="input_text" autofocus onfocus="this.select()" value="copied text"></form>
+    <div id="log"></div>
+    <script>
+    (function(){
+        var t = async_test()
+        var dataToPaste="clipboard text" ;
+        var extPassCond="clipboard text" ;
```

Comment:
Anything with an external pass condition can't be verified through JS and therefore needs to be -manual.